cgi/org.pl

```diff
@@ -358,29 +358,28 @@
 	if ($type eq "edit") {
 
 		store_org($org_ref);
-		$template_data_ref->{result} = lang("edit_org_result");
-
-		$template_data_ref->{profile_url} = canonicalize_tag_link("editors", "org-" . $orgid);
-		$template_data_ref->{profile_name} = sprintf(lang('user_s_page'), $org_ref->{name});
 	}
 	elsif ($type eq 'user_delete') {
 
 		if (is_user_in_org_group($org_ref, $User_id, "members")) {
```

Comment:
Only admins should be able to remove other users !

```perl
        if (is_user_in_org_group($org_ref, $User_id, "admins")) {
```

Reply:
Note that currently all org members are org admins, we'll need to address that in future PRs.